### DISCUSSION TITLE
Source CAD files

### DISCUSSION THREAD

**Discussion by User-462** [srozum/film_camera_tester#37] [General]
@srozum Would you consider uploading the source CAD files (Solidworks, STEP, etc.) for the project?
It would be useful to move or add holes and make other adjustments. For example, on the Light-unit the holes on the Flexible Hood do not align with the holes on the Face-panel and Reflector-mount. 
Also, it would be helpful to add small pilot holes in several areas of the CAD model. This is because any hole in the CAD model gets a circular reinforcement in the slicing process, which is more robust to drill and/or tap. 
On the light sensor faceplates there are cases where the internal print has a partial infill that can interfere with drilling a clean hole, and possibly light passing to the sensor diodes through the tiny holes. An undersized hole in the model may not print perfectly, but it will machine cleanly.
Naturally any modifications would be available for addition to this repository if you approve them.
![faceplate infill Screenshot 2023-09-01 135658](https://github.com/srozum/film_camera_tester/assets/143836434/a19ae49c-80f4-48cb-925e-7951ac12ffeb)

**Comment by 1N4003** [comment]
Sounds great!
